entrypoint.sh

```diff
@@ -84,7 +89,7 @@ else
 		/.github export-ignore
 		EOL
 
-		# Ensure we are in the $GITHUB_WORKSPACE directory, just in case
+		# Ensure we are in the $WORKSPACE_DIR directory, just in case
 		# The .gitattributes file has to be committed to be used
 		# Just don't push it to the origin repo :)
 		git add .gitattributes && git commit -m "Add .gitattributes file"
```

Comment:
```suggestion
		# Ensure we are in the github repository directory tree. (just in case)
		# The .gitattributes file has to be committed to be used
		# Just don't push it to the origin repo :)
		git add $GITHUB_WORKSPACE/.gitattributes && git commit -m "Add .gitattributes file"
```